src/test/java/core/basesyntax/model/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,88 @@
+package core.basesyntax.model;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.excepction.UserRegistrationException;
+import core.basesyntax.service.RegistrationServiceImpl;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static RegistrationServiceImpl registrationService;
+    private static User user;
+    private static final String VALID_LOGIN = "Volodymyr";
+    private static final String INVALID_LOGIN = "abcde";
+    private static final int VALID_AGE = 18;
+    private static final int INVALID_AGE = 17;
+    private static final String VALID_PASSWORD = "Password";
+    private static final String IVALID_PASSWORD = "Badpa";
+
+    @BeforeAll
+    static void beforeAll() {
+        registrationService = new RegistrationServiceImpl();
+    }
+
+    @BeforeEach
+    void setUp() {
+        registrationService = new RegistrationServiceImpl();
+        user = new User(VALID_LOGIN, VALID_PASSWORD, VALID_AGE);
+    }
+
+    @Test
+    void nullUser_notOk() {
```

Comment:
let's follow the next naming convention: `<methodUnderTest>_<state>_<expectedBehavior>`
fix this issue in all places